recipes/qt/6.x.x/conanfile.py

```diff
@@ -967,6 +967,8 @@ def _create_plugin(pluginname, libname, plugintype, requires):
         ]
         self.cpp_info.components["qtCore"].set_property("pkg_config_custom_content", "\n".join(pkg_config_vars))
 
+        if self.settings.build_type == "Release":
+            self.cpp_info.components['qtCore'].defines.append('QT_NO_DEBUG')
```

Comment:
I wonder if it would be worth defining it for `MinSizeRel` too.

Reply:
Yes you are right. What are conan options here, is it same as CMake?

Reply:
Ok, I see settings.yml:
`build_type: [None, Debug, Release, RelWithDebInfo, MinSizeRel]`
I will change the condition
in Qt provided cmake it is:
`set_property(TARGET Qt6::Core APPEND PROPERTY INTERFACE_COMPILE_DEFINITIONS $<$<NOT:$<CONFIG:Debug>>:QT_NO_DEBUG>)`